tests/services/githubevent.service.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { GithubEventService } from '../../src/app/core/services/githubevent.service';
+import { of } from 'rxjs';
+import { EVENTS, ADD_LABEL_EVENT, CHANGE_TITLE_EVENT } from '../constants/githubevent.constants';
+
+let githubService: any;
+let issueService: any;
+
+describe('GithubEventService', () => {
+  beforeAll(() => {
+    githubService = jasmine.createSpyObj('GithubService', ['fetchEventsForRepo']);
+  });
+
+  describe('.setLatestChangeEvent()', () => {
+    it('stores the time of the most recent issue event\ and most recent issue update.', async () => {
+      githubService.fetchEventsForRepo.and.returnValue(of(EVENTS));
+      const githubEventService: GithubEventService = new GithubEventService(githubService, null);
```

Comment:
Is it possible to have this in beforeAll() so that we don't need initialise it in each test case?